commands/bzpopmin.md

```diff
@@ -23,6 +23,10 @@ popped from.
   where a member was popped, the second element is the popped member itself,
   and the third element is the score of the popped element.
 
+@history
+
+* `>= 6.0`: `timeout` is interpreted as a double instead of an integer.
+
```

Comment:
don't we need a comment like that in BZPOPMAX too?
and also in LMOVE and BRPOPLPUSH

Reply:
WRT BZPOPMAX, as it refers to BZPOPMIN for syntax I thought it would suffice, but sure, it can be added.

WRT LMOVE, it doesn't have a timeout and BLMOVE is a new command anyway.

WRT BRPOPLPUSH, same logic as BZs but added.